### DISCUSSION TITLE
Setting up AVR from Rasberry Pi

### DISCUSSION THREAD

**Discussion by KnowTheFuture** [revoxhere/duino-coin#466] [Q&A] [answered]
Hi,

I've had some Arduino boards laying around and bought a Rasberry Pi 4 for my son. It will be a while since he is ready to use it, which offers me a chance to play around too.

The Arduino board is connected to the Pi's USB slot and uploads to the board, but it is not configured properly. AVR_Miner and PC_Miner do not open to make the configuration easy.

For example:
- I see IDString - "DUCOID" on the Arduino_Code tab. 
- 
- Do I need to change these settings manually?

Could someone walk me through the process?

Thank you.

KTF

**Comment by revoxhere** [comment, marked as answer]
No, you don't need to change those. Have you read the [getting started](https://duinocoin.com/getting-started) guide on our website? Just upload the Arduino Code to your board and launch the AVR miner using `python3 AVR_Miner.py`. Everything is explained nicely there and in the README.

**Reply by KnowTheFuture** [reply]
![F2117183-B806-4370-9D7F-5F79373AE897_1_201_a](https://user-images.githubusercontent.com/8218649/117130113-8f162100-adda-11eb-8045-f0386dd102db.jpeg)

Thank you @revoxhere got it to work, but now I am receiving this error that it is not connecting. Probably a port error. How do I reset the config file?

**Reply by revoxhere** [reply]
Hey, please run the miner through the terminal and not IDLE. Also make sure nothing else (serial monitor for example) is using the serial port.
